firmware/application/apps/ui_bht_tx.hpp

```diff
@@ -222,8 +227,8 @@ class BHTView : public View {
 	
 	Checkbox checkbox_cligno {
```

Comment:
Cligno is a short for 'Clignotant' => flashing. Should we adjust the checkbox_name ?

Reply:
Sure I can also change some "internal names"

Reply:
changed some internal names for both programs. Thanks for the feedback.